cli/providers/aws/elb/wait.go

```diff
@@ -9,43 +9,70 @@ import (
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/elb"
+	"github.com/pchico83/i2kit/cli/schemas/service"
 )
 
-//Wait waits for the instances in a ELB to be registered
-func Wait(name string, config *aws.Config, log *logger.Logger) error {
+//Wait waits for the instances in a ELB to be healthy
+func Wait(s *service.Service, name string, config *aws.Config, log *logger.Logger) error {
+	registeredInstances, err := waitForRegistered(s, name, config, log)
+	if err != nil {
+		return err
+	}
+	log.Printf("Waiting for %s to be healthy in the '%s' load balancer...", registeredInstances, name)
+	for _, instanceID := range registeredInstances {
+		if err := waitForHealthy(name, instanceID, config, log); err != nil {
+			return err
+		}
+	}
+	log.Printf("%v are healthy in the '%s' load balancer.", registeredInstances, name)
+	return nil
+}
+
+func waitForRegistered(s *service.Service, name string, config *aws.Config, log *logger.Logger) ([]string, error) {
 	log.Printf("Waiting for instances to be registered in the '%s' load balancer...", name)
+	retries := 0
+	registeredInstances := []string{}
+	for retries < 30 {
+		registeredInstances = Instances(name, config)
+		log.Printf("%v instances registered in the '%s' load balancer.", registeredInstances, name)
```

Comment:
I'd only log this if the instance count changes, to avoid having 10 logs with the same text sent to the user.

Reply:
In general, I prefer to log something every 10 seconds, to show that some progress in going on.
Thoughts?